website/sidebars.json

```diff
@@ -1,10 +1,20 @@
 {
   "docs": {
-    "Docusaurus": ["doc1"],
-    "First Category": ["doc2"],
-    "Second Category": ["doc3"]
-  },
-  "docs-other": {
-    "First Category": ["doc4", "doc5"]
+    "User Manual": [
+      "getting-started",
```

Comment:
Ah,  id:`index` is not here.

Reply:
@azu, thank you for your review. I realized that in order to show sidebar, a document should be listed in `sidebars.json`.

https://github.com/facebook/Docusaurus/blob/bee98053c87c05b2365562e46066833bd3d59879/lib/core/DocsSidebar.js#L19-L22

I temporarily added `index` and it looks redundant—body of the `index` and contents listed at the side bar are identical. How do you think? Any idea?

<img width="721" alt="screen shot 2018-01-15 at 22 15 16" src="https://user-images.githubusercontent.com/679719/34944130-b9fa7682-fa41-11e7-8800-1db1313a136a.png">

Reply:
My thought:

- Remove current "README" as it can be replace with sidebar
- Use "Getting Started with textlint" as index of documentation

Reply:
We can leave README as is (it will be useful when a user see repository at GitHub directory.)

Reply:
Fm.
We can use "Getting Started with textlint" as index.

I propose that 

- Remove `index` from sidebar.json
- Use `getting-started` to headerLinks(`siteConfig.js`)

Reply:
I'll take your proposal. Thanks!